bluepyopt/ephys/morphologies.py

```diff
@@ -50,17 +50,22 @@ def __init__(
             do_replace_axon=False,
             do_set_nseg=True,
             comment='',
-            replace_axon_hoc=None):
+            replace_axon_hoc=None,
+            morph_modifiers=None):
         """Constructor
 
         Args:
             morphology_path (str): location of the file describing the
                 morphology
-            do_replace_axon(bool): Does the axon need to be replaced by an AIS
-                stub ?
-            replace_axon_hoc(str): String replacement for the 'replace_axon'
-            command in hoc  Must include 'proc replace_axon(){ ... }  If None,
-            the default replace_axon is used in any created hoc files
+            do_replace_axon (bool): Does the axon need to be replaced by an AIS
+                stub with default function ?
+            replace_axon_hoc (str): String replacement for the 'replace_axon'
+                command in hoc  Must include 'proc replace_axon(){ ... }
+                If None,the default replace_axon is used
+            do_set_nseg (bool/float): if True, it will use default frequency
```

Comment:
I find this naming a bit confusing. When I see this name 'nseg' I'd assume we're setting the nseg to a fixed value, but this is not the case.

Reply:
this is the code that is in proj82/singlecell-optimisation, I'm happy to modify it to the better. What would you suggest? replace do_set_nseg -> set_nseg ?

Reply:
I'd say nseg_frequency, or nseg_div. What do you think?

Reply:
it uses div in the code, but I would better understand frequency

Reply:
changing that name may also break other codes, that's why I left it as it is

Reply:
I thought the best was to add another argument to provide a nseg_frequency, which will be =40 by default